homeassistant/components/hassmpris/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "hassmpris",
```

Comment:
This is a strange domain name, shouldn't it be `mpris_client` or something like that, since the integration will allow Home Assistant to act as an mpris client?

Reply:
Fair.  Let's see how we can fix this.  Will need a minute.

Reply:
Yup.  Will rename.